packages/sanity/src/core/store/bundles/useBundleOperations.ts

```diff
@@ -0,0 +1,50 @@
+import {uuid} from '@sanity/uuid'
+import {useCallback} from 'react'
+
+import {useAddonDataset} from '../../studio/addonDataset/useAddonDataset'
+import {type BundleDocument} from './types'
+
+// WIP - Raw implementation for initial testing purposes
+export function useBundleOperations() {
+  const {client} = useAddonDataset()
+
+  const handleCreateBundle = useCallback(
+    async (bundle: Partial<BundleDocument>) => {
+      const document = {
```

Comment:
COULD: assign fields like `_createdAt`, `_updatedAt`, `_rev` at this point

Reply:
We could pre populate `_createdAt` and `_updatedAt`, but they will be added automatically by content lake when creating the document.
`_rev` will always be overridden by CL.